audio/nrsc5/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        theori-io nrsc5 f1a32a551471dc5045193027e4c828746c5790fd
+
+revision            0
+
+categories          audio
+maintainers         {netjibbing.com:blake @trodemaster}
```

Ports should also be `openmaintainer` unless there is a very specific reason why you and only you can make modifications to this port:

```suggestion
maintainers         {netjibbing.com:blake @trodemaster} \
                    openmaintainer
```